src/v2/controllers/vreplicaset_controller/trusted/spec_types.rs

```diff
@@ -152,4 +171,35 @@ pub struct VReplicaSetSpecView {
     pub template: Option<PodTemplateSpecView>,
```

Comment:
This should not be Option (my bad for defining it this way). We should use this as reference: https://github.com/Arnavion/k8s-openapi/blob/master/src/v1_30/api/apps/v1/deployment_spec.rs#L28